### ISSUE TITLE
Cannot parse type 19 from file

### ISSUE THREAD

**Issue by IngoGege** [stephenegriffin/mfcmapi#131]
Hi @stephenegriffin ,
I used MrMapi in a few scripts to parse the ACL table dumped into a file:
C:\Temp\MrMAPI.exe -P 19 -Input C:\Temp\Perm1.tmp
The content of the Perm1.tmp is the Binary  converted to Hex value from the extended property [Microsoft.Exchange.WebServices.Data.FolderSchema]::Permissions. When I run MrMapi it returns NULL.
Any thoughts?
Ciao,
Ingo

**Comment by stephenegriffin** [comment]
Perhaps you saved the file as Unicode instead of ascii?

**Comment by glfhern** [comment]
Thank you,
Gina Hern

**Comment by IngoGege** [comment]
@stephenegriffin I used ascii. What kind of input is allowed?

**Comment by stephenegriffin** [comment]
Try using the _Hex Editor_ in MFCMAPI to test your data out. But basically, it should be an even number of hex digits. I do allow for whitespace and a few common punctuation characters (like dot or dash) which show up commonly.

**Comment by IngoGege** [comment]
Hi @stephenegriffin ,
I used MFCMAPI and compared the values: They are equal. Therefore I copied from MFCMAPI the HEX value and saved this into a file, but there is still no output.
Ciao,
Ingo

"08000300000000000100048.....CC3A822967EFD180300" | Out-File -FilePath C:\Temp\Perm2.txt -Encoding Ascii -Force

C:\Temp>MrMAPI.exe -p 19 -i c:\Temp\Perm2.txt -v
Mode = 4
ulOptions = 0x00050200
ulTypeNum = 0xFFFFFFFF
lpszInput = c:\Temp\Perm2.txt
Loading AddIns
ReadStringFromRegistry(AddInExclusionList)
ReadStringFromRegistry(AddInInclusionList)

Current dir = "C:\Temp"
File spec = "C:\Temp\*.dll"
Examining "acctinfo.dll"
Loading default arrays
Found 0x00000F5C built in prop tags.
Found 0x00000023 built in prop types.
Found 0x00000044 built in guids.
Found 0x00000330 built in named ids.
Found 0x000005E2 built in flags.
Found 0x0000008B built in Smart View parsers.
Found 0x00000021 built in Smart View parser types.
Done loading AddIns
CBinaryParser::Init cbBin = 0x00000000 = 0
Unloading AddIns
Done unloading AddIns
